package.json

```diff
@@ -19,35 +18,36 @@
     "access": "public"
   },
   "dependencies": {
-    "chalk": "^2.4.1",
-    "esm": "^3.1.0",
+    "chalk": "^4.0.0",
     "ip": "^1.1.4",
-    "minio": "^6.0.0",
-    "request": "^2.87.0",
-    "request-promise-native": "^1.0.5",
+    "minio": "^7.0.16",
+    "request": "^2.88.2",
+    "request-promise-native": "^1.0.8",
     "rsa-pem-from-mod-exp": "^0.8.4",
-    "winston": "^3.0.0"
+    "winston": "^3.2.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.9.6",
+    "@babel/plugin-transform-runtime": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
     "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.6",
-    "babel-jest": "^23.4.2",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-airbnb": "^2.5.3",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "eslint": "^5.4.0",
-    "eslint-config-airbnb-base": "^13.0.0",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.0.1",
+    "babel-preset-airbnb": "^5.0.0",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-defaults": "^9.0.0",
-    "eslint-config-prettier": "^3.0.1",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jest": "^21.18.0",
-    "eslint-plugin-prettier": "^2.6.2",
-    "jest": "^23.6.0",
-    "prettier": "^1.14.3"
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.13.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "gulp": "^4.0.2",
+    "gulp-babel": "^8.0.0",
+    "gulp-clean": "^0.4.0",
+    "jest": "^26.0.1",
+    "prettier": "^2.0.5",
+    "transform-runtime": "0.0.0"
   },
   "scripts": {
     "build": "NODE_ENV=production gulp",
```

Comment:
i'd suggest adding a `"prepare":` script which can **run npm build** before a publish

Reply:
Done. I added `prepare` to scripts and added the `files:` to packages.json so it knows to publish files in the `dist/**/*` location (and not the root dir).